dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04305UpdateWorkflowActionTable.java

```diff
@@ -0,0 +1,633 @@
+    private static final String MSSQL_SELECT_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "SELECT name FROM sys.indexes WHERE object_id = (SELECT object_id FROM sys.objects WHERE name = 'workflow_scheme_x_structure') AND name = 'workflow_idx_scheme_structure_2'";
+    private static final String ORACLE_SELECT_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "SELECT index_name, table_owner, table_name, uniqueness FROM user_indexes WHERE table_name = 'WORKFLOW_SCHEME_X_STRUCTURE' AND index_name = 'WK_IDX_SCHEME_STR_2'";
+
+    private static final String MYSQL_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "DROP INDEX workflow_idx_scheme_structure_2 ON workflow_scheme_x_structure";
+    private static final String POSTGRES_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "DROP INDEX IF EXISTS workflow_idx_scheme_structure_2 CASCADE";
+    private static final String MSSQL_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = MYSQL_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX;
+    private static final String ORACLE_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "DROP INDEX WK_IDX_SCHEME_STR_2";
+
+    private static final String MYSQL_DROP_WORKFLOW_ACTION__STEP_FK = "alter table workflow_action drop foreign key fooconstraint     DROP INDEX workflow_idx_scheme_structure_2 ON workflow_scheme_x_structure";
+    private static final String POSTGRES_DROP_WORKFLOW_ACTION__STEP_FK = "DROP INDEX IF EXISTS workflow_idx_scheme_structure_2 CASCADE";
+    private static final String MSSQL_DROP_WORKFLOW_ACTION__STEP_FK = MYSQL_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX;
+    private static final String ORACLE_DROP_WORKFLOW_ACTION__STEP_FK = "DROP INDEX WK_IDX_SCHEME_STR_2";
+
+
+
+    private static final String POSTGRES_CREATE_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "CREATE INDEX workflow_idx_scheme_structure_2 ON workflow_scheme_x_structure(structure_id)";
+    private static final String ORACLE_CREATE_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "CREATE INDEX wk_idx_scheme_str_2 ON workflow_scheme_x_structure(structure_id)";
+
+    // FOREIGH KEYS
+    private static final String MYSQL_CREATE_INTERMEDIATE_TABLE_FOREIGN_KEY_ACTION_ID    = "alter table workflow_action_step add constraint fk_workflow_action_step_action_id foreign key (action_id) references workflow_action(id)";
+    private static final String MYSQL_CREATE_INTERMEDIATE_TABLE_FOREIGN_KEYS_STEP_ID     = "alter table workflow_action_step add constraint fk_workflow_action_step_step_id   foreign key (step_id)   references workflow_step  (id)";
+
+    private static final String POSTGRES_CREATE_INTERMEDIATE_TABLE_FOREIGN_KEY_ACTION_ID = "alter table workflow_action_step add constraint fk_workflow_action_step_action_id foreign key (action_id) references workflow_action(id)";
+    private static final String POSTGRES_CREATE_INTERMEDIATE_TABLE_FOREIGN_KEYS_STEP_ID  = "alter table workflow_action_step add constraint fk_workflow_action_step_step_id   foreign key (step_id)   references workflow_step  (id)";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'MYSQL_CREATE_INTERMEDIATE_TABLE_FOREIGN_KEYS_STEP_ID' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 230 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)